File: tests/Fortran/while_loop.f90
program factorial  
   implicit none  

   ! define variables
   integer :: nfact
   integer :: n 
   n = 1 
   nfact = 1 
   
   ! compute factorials   
   do while (n <= 10)       
      nfact = nfact * n 
      n = n + 1
      print*,  n, " ", nfact   
   end do


   test % test

end program factorial

File: tests/C/while_loop.c
// factorial
void main(void){

	// define variables
	int nfact;
	int n;
	n = 1;
	nfact = 1;

	// compute factorials
	while (n <= 10){
		nfact = nfact * n;
		n = n + 1;
		printf("%d %s %d ", n, " ", nfact);
	}
}